### DISCUSSION TITLE
Revisiting the old issue of impossibility of having coloured text in Zettlr

### DISCUSSION THREAD

**Discussion by SN-CH** [Zettlr/Zettlr#4200] [Q&A]
This question have already been asked many times on Reddit and Github (see, for example, https://github.com/Zettlr/Zettlr/issues/555). The answer was «There is no way coulored text can be implemented». I realise the rationale behind this renouncement, the software is designed to be fully compatible with standard Markdown. Another possible reason is the lack of enthusiasm on part of developers to blow up Zettlr into another rtf or html WYSIWIG editor.

But times change and maybe there is something that can be done here (as an exclusion, possibly).

It is great that Zettlr plays well with Pandoc (I regularly export to LaTeX). But, I know for a fact that some people don't have much use for this export mechanism. Actually, they prefer to use Zettlr to store textual information, in other words as a database of interlinked and tagged notes (after all, this is a software aimed at people who wish to organise their work in accordance with the Zettelkasten principles, is it?). And Zettlr is pretty good at that. That being said, possibility to colorise some pieces of text could come in handy in such case (yes, one can use bold and italic to have distinguished look for certain pieces of text, but... the idea of mixing different character styles in one document does not appeal to every one).

**Comment by exanrauzer** [comment]
> Actually, they prefer to use Zettlr to store textual information, in other words as a database of interlinked and tagged notes (after all, this is a software aimed at people who wish to organise their work in accordance with the Zettelkasten principles, is it?). And Zettlr is pretty good at that.

I agree. Zettlr lacks in highlighting. But I would rather opt for a proper rendering of markdown based marking. For now simple ==text== lacks proper rendering:

![image](https://github.com/Zettlr/Zettlr/assets/103963891/be29af73-60eb-4e7a-8033-9f1391bb1721)

I would suggest focusing on proper and full utilization of Markdown's potential.
